exchanges/bitfinex/bitfinex_test.go

```diff
@@ -157,18 +157,13 @@ func TestUpdateOrderExecutionLimits(t *testing.T) {
 		},
 	}
 	for assetItem, pairs := range tests {
-		if err := b.UpdateOrderExecutionLimits(t.Context(), assetItem); err != nil {
-			t.Errorf("Error fetching %s pairs for test: %v", assetItem, err)
+		if assert.NoError(t, b.UpdateOrderExecutionLimits(t.Context(), assetItem), "UpdateOrderExecutionLimits should not error") {
 			continue
 		}
 		for _, pair := range pairs {
-			limits, err := b.GetOrderExecutionLimits(assetItem, pair)
-			if err != nil {
-				t.Errorf("GetOrderExecutionLimits() error during TestExecutionLimits; Asset: %s Pair: %s Err: %v", assetItem, pair, err)
-				continue
-			}
-			if limits.MinimumBaseAmount == 0 {
-				t.Errorf("UpdateOrderExecutionLimits empty minimum base amount; Pair: %s Expected Limit: %v", pair, limits.MinimumBaseAmount)
+			l, err := b.GetOrderExecutionLimits(assetItem, pair)
+			if !assert.Error(t, err, "GetOrderExecutionLimits should not error") {
+				assert.NotZero(t, l.MinimumBaseAmount, "GetOrderExecutionLimits should not return empty minimum base amount")
```

Comment:
🚧 I think this is incorrect, in that it's using `!assert.Error` when we're wanting no error. That's asserting it *will* error, and then running a check if that passes
🐒 Also, I think we're using `Get...Limits` to verify that `Update` worked, so the failure wording should be around setting something, not around the method we use to check it.
```suggestion
			require.Error(t, err, "GetOrderExecutionLimits must not error") {
			assert.NotZero(t, l.MinimumBaseAmount, "MinimumBaseAmount should be set")
```